lib/internal/abort_controller.js

```diff
@@ -166,8 +185,10 @@ class AbortSignal extends EventTarget {
    * @param {any} [reason]
    * @returns {AbortSignal}
    */
-  static abort(
-    reason = new DOMException('This operation was aborted', 'AbortError')) {
+  static abort(reason) {
+    if (reason === undefined) {
+      reason = lazyDOMException();
+    }
```

Comment:
Would `reason ??= lazyDOMException();` also work in here, and in the next usage?

Reply:
I think so yes! updating this (since benchmark ci is running would pushing a commit interfere with it?)

Reply:
No, it won't interfere with it. 👍

Reply:
Done!

Reply:
I don’t think this is correct, users should be able to supply `null` as a reason

Reply:
Oh ok yes reverting